packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -1203,4 +1203,102 @@ void main() {
       await testColor(color);
     }
   });
+  testWidgets('RefreshIndicator default elevation value correctly',
+    (WidgetTester tester) async {
+  const List<double> positions = <double>[50.0, 100.0, 150.0];
+
+    final AnimationController positionController =
+        AnimationController(vsync: const TestVSync());
```

Comment:
```suggestion
    final AnimationController positionController = AnimationController(vsync: const TestVSync());
```
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#prefer-a-maximum-line-length-of-80-characters